npm/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "flow-typed",
+  "version": "2.0.0",
+  "description": "A repository of high quality flowtype definitions",
+  "main": "dist/index.js",
+  "bin": "dist/cli.js",
+  "scripts": {
+    "flow": "flow; [[ $?=0||$?=2 ]]",
+    "watch": "mkdir -p dist; babel --watch=./src --out-dir=./dist",
+    "build": "mkdir -p dist; babel ./src --out-dir=./dist",
+    "prepublish": "mkdir -p dist; babel ./src --out-dir=./dist"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/splodingsocks/FlowTyped.git"
```

Comment:
No I think I just don't have push access to your branch. I was going to edit the file directly on Github. I have been able to merge PRs, though, so I think I have write access to the repo.